src/sqlfluff/core/linter/linted_file.py

```diff
@@ -349,12 +349,12 @@ def fix_string(self) -> Tuple[Any, bool]:
                 filtered_source_patches.append(enriched_patch)
                 dedupe_buffer.append(enriched_patch.dedupe_tuple())
             # If it's ONLY templated then we should skip it.
-            elif "literal" not in local_type_list:
+            elif "literal" not in local_type_list:  # pragma: no cover
```

Comment:
A couple areas in this file became "uncovered" because the upstream `has_template_conflicts()` check prevents hitting this code. We could potentially remove it, but TBH, I'd prefer leaving it in place, at least for now, as a second line of defense. (I'm not sure `has_template_conflicts()` eliminates all possibilities of hitting this code.)